notebooks/single_baseline_postprocessing_and_pspec.ipynb

```diff
@@ -551,7 +551,7 @@
     "xtalk_overlaps = {}\n",
```

Comment:
<p>Can you update this "Note that" since it's no longer true?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>